dracut/30ignition/module-setup.sh

```diff
@@ -59,6 +59,9 @@ install() {
     inst_script "$moddir/ignition-setup-user.sh" \
         "/usr/sbin/ignition-setup-user"
 
+    inst_script "$moddir/ignition-kargs-helper.sh" \
+        "/usr/bin/ignition-kargs-helper"
```

Comment:
`/usr/bin` is inconsistent with `ignition-setup-user`, but maybe less confusing to distro maintainers?

Reply:
Switched to `sbin` for consistency.